payments-core/src/main/java/com/stripe/android/model/parsers/BankConnectionsResourceLinkAccountSessionJsonParser.kt

```diff
@@ -0,0 +1,20 @@
+package com.stripe.android.model.parsers
+
+import com.stripe.android.core.model.StripeJsonUtils
+import com.stripe.android.core.model.parsers.ModelJsonParser
+import com.stripe.android.model.LinkAccountSession
+import org.json.JSONObject
+
+internal class BankConnectionsResourceLinkAccountSessionJsonParser : ModelJsonParser<LinkAccountSession> {
+    override fun parse(json: JSONObject): LinkAccountSession {
+        return LinkAccountSession(
+            clientSecret = StripeJsonUtils.optString(json, FIELD_CLIENT_SECRET)!!,
```

Comment:
Can we avoid the use of `!!`, could throw an exception here or use a default value?

Reply:
made fields nullable - https://github.com/stripe/stripe-android/pull/4711/commits/7ed44491339bbd3e8e5304a199843ede1223225a

Will require those on the domain layer once we build the collect bank account launcher.